mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -2384,12 +2384,29 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.SecurityPostureConfig = expandSecurityPostureConfig(v)
 	}
 
+  needUpdateAfterCreate := false
```

Comment:
Hmm, I tabified but it seems to have affected quite a bit of the file. WDYT?

Reply:
Thanks for updating the file. Sorry I was not clear before. I mean let's update the indentation of the code changed in this PR but not the whole file. Sorry about it. Plus, we use spaces for inline spacing and tabs for indentation for resource.go file and in test.go file, we use spaces for resource configuration blocks in the tests: https://github.com/mattcary/magic-modules/blob/pdcsi-create/mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb#L5134.


Sorry if I've made it more confused, you can check generated resource and test files. https://github.com/hashicorp/terraform-provider-google-beta/blob/main/google-beta/services/cloudrunv2/resource_cloud_run_v2_job.go
https://github.com/hashicorp/terraform-provider-google-beta/blob/main/google-beta/services/cloudrunv2/resource_cloud_run_v2_job_generated_test.go
Handwritten files should also follow that format.

Anyways, let's only update the code touched in this PR - this is to prevent merge conflicts for this and other ongoing PRs. Thank you and sorry for the confusion.

Reply:
Okay, I think I got it. I'm not totally sure how to check that tabs are in the right places though.